src/plasoscaffolder/bll/services/sqlite_generator.py

```diff
@@ -1,17 +1,22 @@
 # -*- coding: utf-8 -*-
 """Module representing the sqlite plugin generator"""
+import os
+
+from plasoscaffolder.bll.mappings.init_mapping import *
+from plasoscaffolder.bll.services.file_handler import FileHandler
 from plasoscaffolder.bll.services.sqlite_plugin_helper import *
 
 
 class SqliteGenerator():
   """ Generator for SQLite Files """
 
-  def __init__(self, path, name, database, output):
+  def __init__(self, path: os.path, name: str, database: os.path, output):
     """ Constructs the SQLite Generator
```

Comment:
Contextual: Technically `__init__` is not the constructor but the initializer
https://docs.python.org/3/reference/datamodel.html#basic-customization

Hence I would suggest:
```
Initializes a SQLite Generator.
```